jte-models/src/main/java/gg/jte/models/generator/ModelConfig.java

```diff
@@ -18,6 +18,17 @@ public String implementationAnnotation() {
         return map.getOrDefault("implementationAnnotation", "");
     }
 
+    public Language language() {
+        String configuredLanguage = map.getOrDefault("language", "JAVA").toUpperCase();
+        Language language = Language.JAVA;
+        try {
+            language = Language.valueOf(configuredLanguage);
+        } catch (IllegalArgumentException ex) {
+            // how to report wrong conciguration? fail or default to Java?
```

Comment:
@casid, I'm unsure how (or even if) to do this. It is an open question for this PR.

Reply:
In my opinion you should fail the build in this case. Building Java when a different language was specified is not what the user wants.

I think throwing an exception with a good description would be good enough, but you should test that.

Reply:
Done in 440c1f0.